Competition2017/src/competition/subsystems/shooter_wheel/commands/UnjamShooterWheelCommand.java

```diff
@@ -0,0 +1,33 @@
+package competition.subsystems.shooter_wheel.commands;
+
+import com.google.inject.Inject;
+
+import competition.subsystems.shooter_wheel.ShooterWheelsManagerSubsystem;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.XPropertyManager;
+
+public class UnjamShooterWheelCommand extends BaseShooterWheelCommand {
+
+    private DoubleProperty threshold;
+
+    @Inject
+    public UnjamShooterWheelCommand(ShooterWheelsManagerSubsystem shooterWheelsManagerSubsystem, XPropertyManager propMan) {
+        super(shooterWheelsManagerSubsystem.getLeftShooter());
```

Comment:
That what I thought because it was originally getting the left shooter so I just 
assumed just to use the left.